docs/faq/code-analysis/does-codacy-check-for-dependencies.md

```diff
@@ -1,5 +1,5 @@
 # Does Codacy check for dependencies?
 
-Codacy doesn't check for dependencies. Codacy performs static code analysis on the source code and checks for code style and security issues, duplication, complexity, and coverage.
+Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as issues in your Codacy dashboard.
 
-To learn more on how to extend analysis on your repositories, see Codacy's [supported languages and tools](../../getting-started/supported-languages-and-tools.md).
+For a list of supported languages and manifest files, see Codacy's [supported languages and tools](../../getting-started/supported-languages-and-tools.md).
```

Comment:
For a better connection between both content, I would try to fit the column title **Dependency vulnerability scanning** on this sentence...  

Would this work? 🤔
Is it important to mention the manifest files?

```suggestion
For a list of supported languages and dependency vulnerability scanning tools and manifest files, see Codacy's [supported languages and tools](../../getting-started/supported-languages-and-tools.md).
```

Reply:
Good point. I'd just reorder the sentence, since "language" and "manifest file" are both objects of the action (what's scanned) and "scanning tools" is what does the scanning. I'm not a fan of those stacked modifiers before tools ("dependency vulnerability scanning tools" is a bit hard to read), but it makes sense for the users who follow the link.

```suggestion
For a list of supported languages and manifest files scanned by the Codacy dependency vulnerability scanning tools, see [Supported languages and tools](../../getting-started/supported-languages-and-tools.md).
```

Reply:
Looks good!